include/crm/crm.h

```diff
@@ -66,7 +66,7 @@ extern "C" {
  * >=3.0.13: Fail counts include operation name and interval
  * >=3.2.0:  DC supports PCMK_EXEC_INVALID and PCMK_EXEC_NOT_CONNECTED
  */
-#  define CRM_FEATURE_SET		"3.13.1"
+#  define CRM_FEATURE_SET		"3.14.0"
```

Comment:
We only need to bump the minor-minor for tool changes (the minor prevents mixed-version clusters except in a rolling upgrade)